Makefile

```diff
@@ -1,6 +1,7 @@
 SDCC ?= sdcc
 STCCODESIZE ?= 4089
-SDCCOPTS ?= --iram-size 256 --code-size $(STCCODESIZE) --xram-size 0 --data-loc 0x30 --disable-warning 126 --disable-warning 59
+SDCCOPTS ?= --iram-size 256 --code-size $(STCCODESIZE) --xram-size 0 --data-loc 0x30 --disable-warning 126 --disable-warning 59 \
+    -DDEBUG -DWITH_ALT_LED9 -DWITHOUT_LEDTABLE_RELOC
```

Comment:
Are/should these be exposed as make build options so user can do something like `make WITH_ALT_LED9=1` etc?
Another option for the future might be to consider individual make targets at least for the board/model variations, and leverage the make build options for various features.
Anyhow, going to merge this now, as it seems to work fine as is on my stc15f204ea version (with two buttons).